howto/monitor/landing.md

```diff
@@ -1,34 +1,7 @@
+[LXD Metric exporter for instances](https://discuss.linuxcontainers.org/t/lxd-metric-exporter-for-instances/11735) has a list of metrics available through LXD.
 
 The implementation of a monitoring or observability solution depends on your specific use case and the tools that you want to use. See [Open source observability for enterprises](https://ubuntu.com/observability) for an overview of observability tools and stacks that are recommended and supported by Canonical. In particular, see the [Canonical Observability Stack (LMA2)](https://juju.is/docs/lma2).
 
-Anbox Cloud includes the following reference implementations that you can use as a starting point for planning and integrating your custom observability solution:
-
-- [Example: Collect metrics](https://discourse.ubuntu.com/t/monitoring-grafana/17787)
-- [Example: Monitor status](https://discourse.ubuntu.com/t/monitoring-nagios/17788)
-
-These examples show how to set up an observability solution in a full Anbox Cloud deployment. They can also be used for a first test. Be aware though that these examples are provided for reference only. They are not fully supported and should not be used in a production environment. They cannot be used with the Anbox Cloud Appliance.
-
-<a name="monitoring-dashboard"></a>
-## Monitoring dashboard in the Anbox Cloud Appliance
-
-The Anbox Cloud Appliance comes with a basic monitoring dashboard that displays the metrics collected through Telegraf and Prometheus.
-
-To access the dashboard, complete the following steps:
-
-1. Log on to the machine that runs the Anbox Cloud Appliance and create a user account for the monitoring dashboard:
-
-        anbox-cloud-appliance monitor account create <user_name> --email=<email_address> --password=<password> --role=<role>
-
-    The available roles are `viewer`, `editor` and `admin`.
-
-2. Make sure that the monitoring dashboard is exposed. This is the default. If you unexposed it earlier, expose it again by running the following command:
-
-        anbox-cloud-appliance monitor expose
-3. Access the monitoring dashboard at `https://<your-machine-address>/monitor/` and log on with the user account that you created.
-4. Click the search symbol on the left-hand side and go to the **Search dashboards** page.
-5. Select the dashboard that contains the information you're interested in to display it.
-
-If you want to monitor additional metrics and events, you can do so by adding plugins to Telegraf. See [How to configure Telegraf](https://discourse.ubuntu.com/t/how-to-configure-telegraf/30365) for more information.
+There are plans in the Anbox Cloud roadmap to provide better monitoring abilities by using the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack).
```

Comment:
It is a bit tricky. I worded it this way because this is just a reassurance because we are pulling out the ability to monitor totally (although some of the stuff like telegraf and nagios still work but they require additional processing which is tied to Grafana or Prometheus. So we are just pulling out everything from the documentation. But that kind of means that there is no supported monitoring solution for Anbox Cloud in the 1.20 release. So this statement it to reassure the users that we are working on it. I will use your rewording once I add the corresponding version when we are targeting to release integration with COS. Hope that is okay?

Reply:
I'm not sure if my version is better than yours, I just wanted to point out the perceived writing style difference.
Thanks for the explanation!